src/credential/credentialExpiryHandler.ts

```diff
@@ -25,6 +25,11 @@ export class CredentialExpiryHandler {
 
     public async userCredentialRefreshRequestHandler(params: GetCredentialParams): Promise<GetCredentialResponse> {
         return new Promise<GetCredentialResponse>(async (resolve, reject) => {
+            if (this._environmentController.userType === UserType.PublicContributor) {
+                reject(new Error(`Some required data to validate this repository is only accessible to Microsoft employee. ` +
```

Comment:
what does user see when you directly reject it? extension crash or a error message window